### PR TITLE
Add Nextbit Robin (ether)

### PR DESCRIPTION
**Pull request by pajlow**

### CODE REVIEW

File: manifests/nextbit_ether.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+	<remote name="nex"
+            fetch="https://github.com/pajlow"
+            revision="cm-14.1" />
```
Comment (lnjX) [inline manifests/nextbit_ether.xml]: Please use revision/branch `halium-7.1` for all your changes.

File: manifests/nextbit_ether.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+<manifest>
+	<remote name="nex"
+            fetch="https://github.com/pajlow"
+            revision="cm-14.1" />
+	
+	<project path="device/nextbit/ether" name="android_device_nextbit_ether" remote="los"/>
+
+	<project path="kernel/nextbit/msm8992" name="android_kernel_nextbit_msm8992" remote="nex" />
+
+	<!-- <project path="device/qcom/common" name="android_device_qcom_common" remote="nex" /> -->
```
Comment (lnjX) [inline manifests/nextbit_ether.xml]: qcom common isn't needed (it's already included in the main manifest)

File: manifests/nextbit_ether.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+	<project path="vendor/ether" name="proprietary_vendor_nextbit" remote="nex" />
```
Comment (lnjX) [inline manifests/nextbit_ether.xml]: Don't fork the vendor repositories!

Comment (jbruechert) [reply manifests/nextbit_ether.xml]: Additionally this repository doesn't seem to actually exist, so you can safely switch to the "them" remote.

File: manifests/nextbit_ether.xml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+	<remote name="nex"
+            fetch="https://github.com/pajlow"
+            revision="halium-7.1" />
+	
+	<project path="device/nextbit/ether" name="android_device_nextbit_ether" remote="los"/>
+
+	<project path="kernel/nextbit/msm8992" name="android_kernel_nextbit_msm8992" remote="nex" />
```
Comment (jbruechert) [inline manifests/nextbit_ether.xml]: Sorry for the additional change request, but it doesn't seem like the halium-7.1 branch actually exists on your repository. Also I couldn't find your kernel config changes there yet.

Comment (pajlow) [reply manifests/nextbit_ether.xml]: No problem. I actually had no changes there.

File: manifests/nextbit_ether.xml [hunk 5]
```diff
@@ -1,12 +1,8 @@
...
-<manifest>
-	<remote name="nex"
-            fetch="https://github.com/pajlow"
-            revision="halium-7.1" />
-	
+<manifest>	
 	<project path="device/nextbit/ether" name="android_device_nextbit_ether" remote="los"/>
 
-	<project path="kernel/nextbit/msm8992" name="android_kernel_nextbit_msm8992" remote="nex" />
+	<project path="kernel/nextbit/msm8992" name="android_kernel_nextbit_msm8992" remote="los" />
```
Comment (jbruechert) [inline manifests/nextbit_ether.xml]: Sorry, but this actually doesn't make it better. You need to change the kernel config and to publish it you need to use a fork of the kernel repository.

**Comment by vanyasem** [comment]
Closing because #110 was merged

**Pull request closed without merging**